src/librustc_driver/pretty.rs

```diff
@@ -425,8 +425,8 @@ impl<'hir> pprust_hir::PpAnn for IdentifiedAnnotation<'hir> {
             }
             pprust_hir::AnnNode::Expr(expr) => {
                 s.s.space()?;
-                s.synth_comment(format!("node_id: {} hir local_id: {}",
-                                        expr.id, expr.hir_id.local_id.as_u32()))?;
+                s.synth_comment(format!("expr hir_id: {} hir local_id: {}",
```

Comment:
You added `expr ` here. Not sure if that's good or bad =P

Reply:
Yes, so that it matches the other branches; I mean, it shouldn't break anything ^^.